build/walk.go

```diff
@@ -145,6 +145,13 @@ func WalkOnce(v Expr, f func(x *Expr)) {
 		f(&v.Then)
 		f(&v.Test)
 		f(&v.Else)
+	case *LoadStmt:
+		for i := range v.From {
```

Comment:
Also need to visit `v.Module`. That's actually the main use case for me: I have a program that needs to perform a substitution on all labels in a file. It works by calling `build.Edit` and matching `*StringExpr`.

Reply:
Yes, you're right, will fix that.

Reply:
Done.